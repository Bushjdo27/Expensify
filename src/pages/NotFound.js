import React, {Component} from 'react';
import { Link } from 'react-router-dom'
const NotFound = () => {
    return (
        <main>
            <h1>
                Page Not Found
            </h1>
            <Link to="/">
                Go Home
            </Link>
        </main>
    )
}


export default NotFound;