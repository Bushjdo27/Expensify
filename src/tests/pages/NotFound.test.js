import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../pages/NotFound';


test('should render NotFound page', ()=>{
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
    
})