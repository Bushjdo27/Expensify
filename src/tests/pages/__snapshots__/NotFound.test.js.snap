// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFound page 1`] = `
<main>
  <h1>
    Page Not Found
  </h1>
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</main>
`;
